python/SWESimulators/CDKLM16.py

```diff
@@ -195,7 +198,7 @@ def cleanUp(self):
         gc.collect()
            
     @classmethod
-    def fromfilename(cls, cl_ctx, filename, cont_write_netcdf=True):
+    def fromfilename(cls, filename, gpu_ctx, cont_write_netcdf=True):
```

Comment:
This seems like a poor redesign from my perspective - should we keep argument order?

Also, what is cls?

Reply:
Agreed. Will change.

"cls" is for class. (Seems to be Python "best practice" for fromfilename-methods.)